apps/heft/src/plugins/SassTypingsPlugin/SassTypingsGenerator.ts

```diff
@@ -110,6 +119,10 @@ export class SassTypingsGenerator extends StringValuesTypingsGenerator {
     });
   }
 
+  private _isSassPartial(filePath: string): boolean {
```

Comment:
Can you include a doc comment describing what a sass partial is?

Reply:
@iclanton Hello Ian, I think I've addressed all your feedback.